src/app/components/multiselect/multiselect.ts

```diff
@@ -1471,6 +1471,10 @@ export class MultiSelect implements OnInit, AfterViewInit, AfterContentInit, Aft
     }
 
     isSelectionAllDisabled() {
+        if (this.showToggleAll && isNaN(this.selectionLimit)) {
```

Comment:
Hi @dfernandez-elastic,

Thanks for the effort and support!

The best solution would be to update the input transformer instead of adding NaN checks to methods.

You can directly implement this solution to your PR.
`@Input({ transform: (value: unknown) => numberAttribute(value, null) }) selectionLimit: number | null | undefined;`

Reply:
Hi @cetincakiroglu it worked! I tested manually for a possible regression, too. Everything works as expected. Thanks for the tip!